week2/2-website/css/style.css

```diff
@@ -5,6 +5,194 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+
+#logo {
+    display: flex;
+    justify-content: center;
+    width: 43vw;
```

Comment:
No need to set a percentage width on these elements.
Allow the contents to dictate the width. This way you'll never take up more space than you need.